src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "Y" : " ");
```

Comment:
This ternary statement could be rewritten to match the CS2103 Java Coding Standard.

From the CS2103 Java Coding Standard (https://se-education.org/guides/conventions/java/intermediate.html)
Here are two acceptable ways to format ternary expressions:
```

alpha = (aLongBooleanExpression) ? beta : gamma;
alpha = (aLongBooleanExpression)
        ? beta
        : gamma;

```

For example, perhaps the following might follow the convention more closely:
``` return (isDone) ? "Y" : " ";```

Reply:
The ternary statement in the code matches the first acceptable way (that the whole ternary expression is in one line). I think it is redundant to include brackets for `isDone`, i.e. `return ((isDone) ? "Y" : " ");` is redundant since the boolean expression is a short one.